server/paymentProviders/transferwise/index.ts

```diff
@@ -159,39 +158,68 @@ async function quoteExpense(
     ),
   };
 
-  if (hasMultiCurrency) {
-    assert(
-      expense.collective.currency === expense.host.currency,
-      'For multi-currency expenses, the host currency must be the same as the collective currency',
-    );
-    assert(
-      expense.currency === targetCurrency,
-      'For multi-currency expenses, the payout currency must be the same as the expense currency',
-    );
-    quoteParams['targetCurrency'] = expense.currency;
-    quoteParams['targetAmount'] = expense.amount / 100;
-  } else if (expense.feesPayer === 'PAYEE') {
-    // Using "else if" because customizing the fee payer is not allowed for multi-currency expenses. See `getCanCustomizeFeesPayer`.
-    assert(
-      expense.host.currency === expense.currency,
-      'For expenses where fees are covered by the payee, the host currency must be the same as the expense currency',
-    );
-    quoteParams['sourceAmount'] = expense.amount / 100;
+  let expenseToPayoutMethodExchangeRate: number | null = null;
+  if (expense.host.data?.useLegacyWiseQuoting === false) {
+    if (expense.feesPayer === 'PAYEE') {
+      // Using "else if" because customizing the fee payer is not allowed for multi-currency expenses. See `getCanCustomizeFeesPayer`.
```

Comment:
We're not using a else if anymore

```suggestion
      // Customizing the fee payer is not allowed for multi-currency expenses. See `getCanCustomizeFeesPayer`.
```